File: test/js/bench_scaled/fib.mjs
// Scaled mirror of test/jacl/bench_scaled/fib.jacl — fib(34) = 5702887.
function fib(n) { return n < 2 ? n : fib(n - 1) + fib(n - 2); }
export function run() { return fib(34); }

File: test/python/bench_scaled/fib.py
"""Scaled mirror of test/jacl/bench_scaled/fib.jacl — fib(34) = 5702887."""
def fib(n):
    if n < 2:
        return n
    return fib(n - 1) + fib(n - 2)
def run():
    return fib(34)
if __name__ == "__main__":
    print(run())
